packages/oui-navbar/src/notification/navbar-notification.html

```diff
@@ -0,0 +1,119 @@
+                'oui-icon-info_circle': menuLink.level === 'info',
+                'oui-icon-warning_circle': menuLink.level === 'warning',
+                'oui-icon-clock-wait': menuLink.level === 'incident'
+            }"
+            aria-hidden="true">
+        </i>
+
+        <!-- Link (href) -->
+        <a class="oui-navbar-notification__content"
+            ng-if="::!!menuLink.url"
+            ng-init="$ctrl.notificationsCreated = $last"
+            ng-click="menuLink.linkClicked(menuLink);$ctrl.navbarCtrl.toggleMenu()"
+            ng-href="{{::menuLink.url}}"
+            oui-navbar-group="{{::$ctrl.menuName}}"
+            oui-navbar-group-last="::$last && !$ctrl.footerTemplate && !$ctrl.footerTitle">
+            <div class="oui-navbar-notification__header">
+                <span class="oui-navbar-notification__title" ng-bind=":: menuLink.subject"></span>
+                <span class="oui-navbar-notification__time" ng-bind="menuLink.time"></span>
+            </div>
+            <div class="oui_navbar-notification__body">
+                <div class="oui-navbar-notification__description" ng-bind-html=":: menuLink.description"></div>
+            </div>
+        </a>
+        <!-- /Link (href) -->
+
+        <!-- Link (ui-sref) -->
+        <a class="oui-navbar-notification__content"
+            ng-if="::!!menuLink.state"
+            ng-init="$ctrl.notificationsCreated = $last"
+            ng-click="menuLink.linkClicked(menuLink);$ctrl.navbarCtrl.toggleMenu()"
+            ui-sref="{{::$ctrl.getFullSref()}}"
+            oui-navbar-group="{{::$ctrl.menuName}}"
+            oui-navbar-group-last="::$last && !$ctrl.footerTemplate && !$ctrl.footerTitle">
+            <div class="oui-navbar-notification__header">
+                <span class="oui-navbar-notification__title" ng-bind=":: menuLink.subject"></span>
+                <span class="oui-navbar-notification__time" ng-bind="menuLink.time"></span>
+            </div>
+            <div class="oui_navbar-notification__body">
+                <div class="oui-navbar-notification__description" ng-bind-html=":: menuLink.description"></div>
+            </div>
+        </a>
+        <!-- /Link (ui-sref) -->
+
+        <button class="oui-navbar-notification__action"
+            type="button"
+            ng-click="menuLink.actionClicked(menuLink)"
+            oui-tooltip="{{::menuLink.isActive ?
+                $ctrl.translations.notification.markRead :
+                $ctrl.translations.notification.markUnread
+            }}">
+            <i class="oui-icon oui-icon-shape-dot"
+                ng-if="menuLink.isActive"
+                aria-hidden="true"></i>
+            <i class="oui-icon oui-icon-shape-circle"
+                ng-if="!menuLink.isActive"
+                aria-hidden="true"></i>
+        </button>
+    </li>
+    <li class="oui-navbar-menu__item oui-navbar-notification oui-navbar-notification-placeholder"
+        ng-if="$ctrl.menuLinks.length === 0">
```

Comment:
`!$ctrl.menuLinks.length` ?

Reply:
Not here. It will be triggered if length is `undefined`, since `$ctrl.menuLinks` can be nullified.